src/Modal.js

```diff
@@ -94,6 +94,12 @@ class Modal extends React.Component {
     /**
      * A callback fired when the escape key, if specified in `keyboard`, is pressed.
      */
+    onEscapeKeyDown: PropTypes.func,
+
+    /**
+     * Support for this function will be depricated. Please use `onEscapeKeyDown` instead
```

Comment:
"deprecated" also can you add the `@deprecated` doclet in the comment (under the description)